app/tests/e2e/test_issuer.py

```diff
@@ -90,6 +89,73 @@ async def test_send_credential(
     assert_that(records).extracting("protocol_version").contains("v1", "v2")
 
 
+@pytest.mark.asyncio
+async def test_create_offer(
```

Comment:
Can we add an e2e tests that creates an offer, creates a connectionless oob invitation, and the other agent accepting the oob inviation, and then waiting for the offer-received webhook event for credentials?

It's quite a complex flow that would be good to have a test for

Reply:
I guess we can

Reply:
isn't there a test for this in oob e2e tests already?

Reply:
ah that one isn't connectionless nvm